src/tests/getPersonaIdentifiers/getPersonaIdentifiers.test.ts

```diff
@@ -0,0 +1,120 @@
+// tslint:disable:no-magic-numbers
+// tslint:disable:max-file-line-count
+import * as assert from 'assert';
+import CreateIdentifierOptions from '../../serviceFactory/options/CreateIdentifierOptions';
+import setup from '../utils/setup';
+import {
+  TEST_ORGANISATION,
+  TEST_PERSONA_ID,
+} from '../utils/values';
+
+describe('getPersonaIdentifiers', () => {
+  const persona = TEST_PERSONA_ID;
+  const service = setup();
+  const identifierData1: CreateIdentifierOptions = {
+    ifi: {
+      key: 'mbox',
+      value: 'mbox:test1@mbox.com',
+    },
+    organisation: TEST_ORGANISATION,
+    persona,
+  };
+  const identifierData2: CreateIdentifierOptions = {
+    ifi: {
+      key: 'mbox',
+      value: 'mbox:test2@mbox.com',
+    },
+    organisation: TEST_ORGANISATION,
+    persona,
+  };
+  const identifierData3: CreateIdentifierOptions = {
+    ifi: {
+      key: 'mbox',
+      value: 'mbox:test3@mbox.com',
+    },
+    organisation: TEST_ORGANISATION,
+    persona,
+  };
+
+  beforeEach(async () => {
+    await service.clearService();
```

Comment:
Again this should be handled by the setup function and not worried about having the clear on after.

Reply:
This is done other test files here too.